server/bin/unittests

```diff
@@ -250,6 +250,10 @@ function _setup_state {
     let res=res+$?
     cp $_tdir/test-bin/* $_testopt_sat/unittest-scripts/
     let res=res+$?
+    cp -a $_tdir/test-lib/* $_testopt/unittest-scripts/
```

Comment:
No sure this should be in `unittest-scripts`; can we consider something like:

```
mkdir ${_testopt}/unittest-lib
cp -a ${_tdir}/test-lib/* ${_testopt}/unittest-lib/
...
```

Reply:
yeah sure